apis/placement/v1beta1/stageupdate_types.go

```diff
@@ -127,7 +127,8 @@ type StageConfig struct {
 
 	// LabelSelector is a label query over all the joined member clusters. Clusters matching the query are selected
 	// for this stage. There cannot be overlapping clusters between stages when the stagedUpdateRun is created.
-	// If the label selector is absent, the stage includes all the selected clusters.
+	// If the label selector is empty, the stage includes all the selected clusters.
+	// If the label selector is nil, the stage does not include any selected clusters.
```

Comment:
what's the corresponding yaml for nil vs empty label selctor?

Reply:
A nil label selector:
```
spec:
  stages:
  - name: staging
```
An empaty label selector:
```
spec:
  stages:
  - labelSelector: {}
    name: staging
```
It's very similar to the `clusterSelector` in overrides.